cmd/syncthing/cli/operations.go

```diff
@@ -38,6 +41,12 @@ var operationCommand = cli.Command{
 			ArgsUsage: "[folder id]",
 			Action:    expects(1, foldersOverride),
 		},
+		{
+			Name:      "default-ignores",
+			Usage:     "Set the default ignores (config) from a file",
+			ArgsUsage: "path",
```

Comment:
@imsodin missed this?

Reply:
Yes I did miss it. I am not a fan of all-caps and tend to avoid it wherever possible. Just had a look at `man man` and they say italic, respectively they actually use underscore. Other man-pages enclose it in `<>`. Anyway, I am consistent with one of the inconsistent options in this PR, so an issue for a different PR?

Reply:
Alright.

Reply:
I just checked again and in `cmd/syncthing/main.go` we use all-caps for arguments consistently.  The documentation has angle brackets throughout.  It's mainly the cli component where it's inconsistent.